### ISSUE TITLE
Difficulty with Win 10 configuration

### ISSUE THREAD

**Issue by quickfold** [jethrokuan/org-roam#492]
### Description

Apologies in advance - I am trying to move to emacs just to use org-roam and cannot solve this myself despite hours of attempts.

I'm on Win 10 and installed Doom Emacs with org-roam. Doom is fine but I can't properly set the org-roam directory! When I launch Doom I get a number of errors ending with "Failed to load 'org' package incrementally, because: (file-missing cannot open load file No such file or directory org-roam)"

The latest attempt that failed was had this in config.el:
(setq org-roam-directory "~/.emacs.d/modules/lang/org/")

- The docs say "Set org-roam-directory to the folder containing all your Org files". Does this mean the folder "C:\Users\Username\.emacs.d\modules\lang\org\"?
- Do I change the slashes from backwards slashes (what I get when copying paths in Win) to forward slashes (which is what is in the instructions)? Or not? I've tried both.

I have set my environmental variables to include HOME at C:\Users\Username. Thanks for any help. It has taken me FOREVER just to get emacs installed on Win and to learn the basics of emacs so I am hoping I can get this working.

### Versions
- Emacs (`C-h v emacs-version`): v26.3, build 1, x86_64-mingw32
- Win 10 x64
- DOOM v2.0

- Org-roam commit: https://github.com/jethrokuan/org-roam/commit/commithashhere

**Comment by Zaeph** [comment]
This seems like a problem with the loading of the `org-roam` package.  Have you downloaded it via MELPA, and if so, have you initialised your packages at the beginning of your config?

**Comment by quickfold** [comment]
Doom Emacs is set up to install org-roam automatically with its package manager by adding a +roam flag to its 'org' line. Earlier, I tried org-roam without specifying any org-roam directory, and it seemed to work, but of course the find files didn't work b/c no directory was set. But I didn't get any errors then upon loading, either.

**Comment by Zaeph** [comment]
Could you paste a minimal config for `org-roam`?  I don't use `doom`, so I don't know its paradigm for requiring packages.

**Comment by jethrokuan** [comment]
> it seemed to work, but of course the find files didn't work b/c no directory was set. 

It defaults to `~/org-roam`. Try setting it to an existing directory. `org-roam-directory` is the directory you want all the created notes to go into, not the code or config for org-roam.

**Comment by quickfold** [comment]
> It defaults to `~/org-roam`. Try setting it to an existing directory. 

I did try setting it to an existing directory first, but it failed. Can you tell me specifically how to do that? As mentioned, I don't even know which way the slashes need to go when setting the directory in Windows. Even though the directory is not supposed to be the module directory, do you know why ``(setq org-roam-directory "~/.emacs.d/modules/lang/org/")`` failed?

The instructions are confusing to a total newbie like me. It says, "The docs say "Set org-roam-directory to the folder containing all your Org files". I assumed that because "Org" is capitalized it meant the module Org. If it means notes, I would reword that. Do the notes in that folder have to have the *.org file extension?

**Comment by quickfold** [comment]
For future searchers, Win 10 does use forward slashes. Check your user accounts environmental variables to make sure you have defined a HOME directory, create a folder at "C:\Users\USERNAME\org-roam\", and put this in your config file:
(setq org-roam-directory "~/org-roam/")
